### PR TITLE
Enable checked CentOS pri 1 build & test jobs per-commit in CI

### PR DESCRIPTION
**Pull request by wtgodbe**
This will enable us to see CentOS pri 1 test results per-commit. We previously had no coreclr test results for CentOS.

### CODE REVIEW

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Comment by wtgodbe** [comment]
@mmitche PTAL

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

File: netci.groovy [hunk 1]
```diff
@@ -912,7 +920,7 @@ combinedScenarios.each { scenario ->
         // Architectures.  x64 only at this point
         ['x64'].each { architecture ->
             // Put the OS's supported for coreclr cross testing here
-            Constants.crossList.each { os ->
```
Comment (mmitche) [inline netci.groovy]: I believe you just want to add Centos to the crossList in the Constants class

Comment (wtgodbe) [inline netci.groovy]: @mmitche I was considering that, but I don't want to, for example, add daily CentOS JIT stress test runs, which adding CentOS to crossList would do (without further refactoring).

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

File: netci.groovy (hunk 1 above)
Comment (mmitche) [inline netci.groovy]: You should keep it in in crossList. You can see below this line where we filter out jobs we don't want to create (for instance, no non Checked builds for stress modes.  You can add additional conditions for OS there.

Comment (wtgodbe) [inline netci.groovy]: Actually I think you're right, see my new change.

**Comment by mmitche** [comment]
There are additional filtration places where you can skip even creating jobs:

```
// Put the OS's supported for coreclr cross testing here
            Constants.crossList.each { os ->
                Constants.configurationList.each { configuration ->

                    def isPR = (branchName == 'pr');

                    if (Constants.jitStressModeScenarios.containsKey(scenario)) {
                        if (configuration != 'Checked') {
                            return
                        }
                        if (isCorefxTesting(scenario)) {
                            return
                        }
                    }
```

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Comment by wtgodbe** [comment]
Gotcha, thanks Matt - I added a filter for CentOS there.

**Comment by mmitche** [comment]
Cool LGTM if the ci test passes.

**Comment by sergiy-k** [comment]
@dotnet-bot test CI please

**Comment by mmitche** [comment]
@sergiy-k You're hitting an assertion in the netci file.  There are a bunch in there to avoid missing conditions:

Assertion failed: 

assert false


	at org.codehaus.groovy.runtime.InvokerHelper.assertFailed(InvokerHelper.java:386)
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.assertFailed(ScriptBytecodeAdapter.java:658)
	at netci$_run_closure2_closure18_closure19_closure20_closure21.doCall(netci.groovy:962)

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Pull request merged**